docs/book/src/user/quick-start.md

```diff
@@ -0,0 +1,477 @@
+```
+
+#### Install the Bootstrap Provider Components
+
+{{#tabs name:"tab-installation-bootstrap" tabs:"Kubeadm"}}
+{{#tab Kubeadm}}
+
+Check the [Kubeadm provider releases](https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases) for an up-to-date components file.
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases/download/v0.1.0/bootstrap-components.yaml
+```
+
+{{#/tab }}
+{{#/tabs }}
+
+
+#### Install Infrastructure Provider Components
+
+{{#tabs name:"tab-installation-infrastructure" tabs:"AWS,vSphere"}}
+{{#tab AWS}}
+
+Check the [AWS provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases) for an up-to-date components file.
+
+For more information about prerequisites, credentials management, and or permissions for AWS, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/docs/getting-started.md).
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v0.4.1/infrastructure-components.yaml
+```
+
+{{#/tab }}
+{{#tab vSphere}}
+
+Check the [vSphere provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases) for an up-to-date components file.
+
+For more information about prerequisites, credentials management, and or permissions for vSphere, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md).
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v0.5.0/infrastructure-components.yaml
+```
+
+{{#/tab }}
+{{#/tabs }}
+
+## Usage
+
+The Cluster API resources are now installed.
+
+![](../images/management-cluster.svg)
+
+Now that we've got Cluster API, Bootstrap and Infrastructure resources installed,
+let's proceed to create a single node cluster.
+
+For the purpose of this tutorial, we'll name our cluster `capi-quickstart`.
+
+{{#tabs name:"tab-usage-cluster-resource" tabs:"AWS,vSphere"}}
+{{#tab AWS}}
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  name: capi-quickstart
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: AWSCluster
+    name: capi-quickstart
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSCluster
+metadata:
+  name: capi-quickstart
+spec:
+  region: us-east-1 # Change this value to the region you want to deploy the cluster in.
+  sshKeyName: default # Change this value to a valid SSH Key Pair present in your AWS Account.
+```
+{{#/tab }}
+{{#tab vSphere}}
+
+<aside class="note warning">
+
+<h1>Action Required</h1>
+
+These examples include environment variables that you should substitute before creating the resources.
```

Comment:
Should we follow a similar path for the AWS examples as well here?

Reply:
I don't think that's necessary, it'll be a quicker quick start for AWS :D

Reply:
unless they are missing the pre-requisites

Reply:
That's fine, we call it out in some places